environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -186,8 +186,14 @@
 
   tasks:
     # NOTE: volume type not available in the openstack.cloud collection
-    - name: Encrypt Volume type named Luks creating
+    - name: Get volume type LUKS
+      ansible.builtin.command: openstack --os-cloud admin volume type show LUKS  # noqa 301
+      register: result
+      ignore_errors: true
```

Comment:
Not sure, but probably a changed_when: false is also required here. Instead of ignore_errors.